relay-general/src/protocol/replay.rs

```diff
@@ -539,6 +554,35 @@ mod tests {
         assert_eq!(maybe_credit_card, Some("[Filtered]"));
     }
 
+    #[test]
+    fn test_capped_values() {
+        let urls: Vec<Annotated<String>> = (0..101)
+            .map(|_| Annotated::new("localhost:9000".to_string()))
+            .collect();
+
+        let error_ids: Vec<Annotated<String>> = (0..101)
+            .map(|_| Annotated::new("52df9022835246eeb317dbd739ccd059".to_string()))
+            .collect();
+
+        let trace_ids: Vec<Annotated<String>> = (0..101)
+            .map(|_| Annotated::new("52df9022835246eeb317dbd739ccd059".to_string()))
+            .collect();
+
+        let mut replay = Annotated::new(Replay {
+            urls: Annotated::new(urls),
+            error_ids: Annotated::new(error_ids),
+            trace_ids: Annotated::new(trace_ids),
+            ..Default::default()
+        });
+
+        let replay_value = replay.value_mut().as_mut().unwrap();
+        replay_value.normalize_array_fields();
+
+        assert!(replay_value.error_ids.value().unwrap().len() == 100);
```

Comment:
Worth also checking for `99` and making sure ur func isn't messing up with that (testing the "other direction")